nilearn/plotting/surface/_utils.py

```diff
@@ -2,22 +2,29 @@
 
 from warnings import warn
 
+import numpy as np
+
+from nilearn._utils import fill_doc
 from nilearn._utils.helpers import is_matplotlib_installed, is_plotly_installed
+from nilearn.plotting._utils import DEFAULT_ENGINE
 from nilearn.surface import (
     PolyMesh,
     SurfaceImage,
 )
 from nilearn.surface.surface import combine_hemispheres_meshes, get_data
 from nilearn.surface.utils import check_polymesh_equal
 
+DEFAULT_HEMI = "left"
```

Comment:
I've set the value to `'left'` for the moment to see if tests pass. 

When it's set to `'both'` as the previously saved images are not the same, the tests that create and compare images fail. 

I think we can keep this as `'left'` until refactoring `nilearn.plotting.surface` package is finished to make sure that we have not diverged from  previous images. When all is complete, we  can change value to `'both'`. How does it soound?